packages/main/src/DatePicker.js

```diff
@@ -476,6 +480,10 @@ class DatePicker extends DateComponentBase {
 			}
 		}
 
+		if ((this._getInput().isEqualNode(event.target) && this.isOpen()) && (isTabNext(event) || isTabPrevious(event)) && (isF6Next(event) || isF6Previous(event))) {
```

Comment:
Shouldn't we treat all those interactions the same way: isTabNext(event) || isTabPrevious(event)) || isF6Next(event) || isF6Previous(event)
If either of those interactions is triggered then close the popover.

Reply:
Yes, that's correct. It was a mistake while copying. Will fix it.